buildifier/buildifier.bzl

```diff
@@ -34,8 +34,25 @@ def buildifier(**kwargs):
 def _buildifier_test_impl(ctx):
     return [buildifier_impl_factory(ctx, test_rule = True)]
 
-buildifier_test = rule(
+_buildifier_test = rule(
     implementation = _buildifier_test_impl,
     attrs = buildifier_attr_factory(True),
     test = True,
 )
+
+def buildifier_test(**kwargs):
+    """
+    Wrapper for the _buildifier_test rule. Optionally disables sandboxing and caching.
+
+    Args:
+      **kwargs: all parameters for _buildifier_test
+    """
+    if kwargs.get("no_sandbox", False):
+        tags = kwargs.get("tags", [])
+
+        # Note: the "external" tag is a workaround for bazelbuild#15516.
```

Comment:
Please use an actual link to the bug, Github-style tags can be confusing for non-Github users (also the repository name is missing there).

Reply:
Done - thanks!